components/login/Form.vue

```diff
@@ -105,30 +66,17 @@ const storeMemberData = () => {
   });
 }
 
-const login = async () => {
-  const url = `${config.public.COMMUNITY_API_URL}/auth/login`;
-  const options = {
+// TODO: separate authentication in a composable
+const login = async() => {
+  await useFetch('/api/login', {
     method: "POST",
     body: JSON.stringify({
       "email": form.email,
       "password": form.password
-    }),
-    headers: {
-      "Accept": "*/*",
-      "Content-Type": "application/json"
-    }
-  }
-  fetch(url, options)
-    .then((response) => response.json())
-    .then((data) => {
-      state.token = data.access_token
-    })
-    .catch((error) => {
-      console.log(error)
-    })
-    .finally(() => {
-      userAuth()
     })
+  })
+  // userAuth()
+  const {data: member} = await useFetch(`${config.public.COMMUNITY_API_URL}/auth/`)
```

Comment:
To fix: session cookies are not sent with this request. See https://stackoverflow.com/a/5422323